app/src/main/java/com/futsch1/medtimer/medicine/AdvancedReminderSettingsFragment.java

```diff
@@ -150,10 +150,12 @@ private void setupRemindOnDays() {
     private void setupCycleStartDate() {
         setCycleStartDate(reminder.cycleStartDay);
         editCycleStartDate.setOnFocusChangeListener((v, hasFocus) -> {
-            if (hasFocus) {
-                TimeHelper.DatePickerWrapper datePickerWrapper = new TimeHelper.DatePickerWrapper(requireActivity().getSupportFragmentManager(), R.string.cycle_start_date);
-                datePickerWrapper.show(getCycleStartDate(), this::setCycleStartDate);
-            }
+            v.post(() -> {
```

Comment:
Please add a comment here (I know there aren't many in the app, but here I feel it's helpful) to explain why this is posted to the view.

Reply:
Added in 2e9749d8ea8f000afe57c4a739f289489efda855